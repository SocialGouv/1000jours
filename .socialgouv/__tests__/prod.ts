//

import { getEnvManifests } from "@socialgouv/kosko-charts/testing";
import { project } from "@socialgouv/kosko-charts/testing/fake/github-actions.env";

jest.setTimeout(1000 * 60);
test("kosko generate --prod", async () => {
  process.env.SOCIALGOUV_PRODUCTION_NAMESPACE = "les1000jours";
  expect(
    await getEnvManifests("prod", "", project("1000jours").prod)
  ).toMatchSnapshot();
});
