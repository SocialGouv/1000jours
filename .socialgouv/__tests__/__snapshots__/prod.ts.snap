// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kosko generate --prod 1`] = `
"---
apiVersion: apps/v1
kind: Deployment
metadata:
  annotations:
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    kapp.k14s.io/delete-strategy: orphan
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/1000jours
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  labels:
    app: strapi-cache
    application: les1000jours
    component: les1000jours
    owner: les1000jours
    team: les1000jours
  name: strapi-cache
  namespace: les1000jours
spec:
  replicas: 1
  selector:
    matchLabels:
      app: strapi-cache
  template:
    metadata:
      annotations:
        kapp.k14s.io/disable-default-ownership-label-rules: ''
        kapp.k14s.io/disable-default-label-scoping-rules: ''
        kapp.k14s.io/delete-strategy: orphan
        app.github.com/job: '5678'
        app.github.com/ref: refs/tags/v1.2.3
        app.github.com/repo: socialgouv/1000jours
        app.github.com/run: '1234'
        app.github.com/sha: '0123456'
      labels:
        app: strapi-cache
        application: les1000jours
        component: les1000jours
        owner: les1000jours
        team: les1000jours
    spec:
      containers:
        - image: nginx:1.19.6
          livenessProbe:
            failureThreshold: 6
            httpGet:
              path: /_health
              port: http
            initialDelaySeconds: 15
            periodSeconds: 5
            timeoutSeconds: 15
          name: strapi-cache
          ports:
            - containerPort: 80
              name: http
          readinessProbe:
            failureThreshold: 15
            httpGet:
              path: /_health
              port: http
            initialDelaySeconds: 5
            periodSeconds: 5
            successThreshold: 1
            timeoutSeconds: 3
          resources:
            limits:
              cpu: '1'
              memory: 1Gi
            requests:
              cpu: 5m
              memory: 256Mi
          startupProbe:
            failureThreshold: 12
            httpGet:
              path: /_health
              port: http
            periodSeconds: 5
            initialDelaySeconds: 0
            timeoutSeconds: 15
          env:
            - name: UPSTREAM
              value: http://strapi
            - name: MAX_SIZE
              value: 1024m
          volumeMounts:
            - mountPath: /var/cache/nginx
              name: strapi-cache
            - mountPath: /etc/nginx/nginx.conf
              subPath: nginx.conf
              name: config
      volumes:
        - name: strapi-cache
          emptyDir: {}
        - name: config
          configMap:
            name: strapi-cache
---
apiVersion: v1
kind: Service
metadata:
  labels:
    app: strapi-cache
    application: les1000jours
    component: les1000jours
    owner: les1000jours
    team: les1000jours
  name: strapi-cache
  annotations:
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    kapp.k14s.io/delete-strategy: orphan
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/1000jours
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  namespace: les1000jours
spec:
  ports:
    - name: http
      port: 80
      targetPort: 80
  selector:
    app: strapi-cache
  type: ClusterIP
---
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  annotations:
    kubernetes.io/ingress.class: nginx
    cert-manager.io/cluster-issuer: letsencrypt-prod
    kubernetes.io/tls-acme: 'true'
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    kapp.k14s.io/delete-strategy: orphan
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/1000jours
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
    nginx.ingress.kubernetes.io/proxy-body-size: 1g
    nginx.ingress.kubernetes.io/limit-rps: '20'
    nginx.ingress.kubernetes.io/limit-rpm: '300'
  labels:
    app: strapi-cache
    application: les1000jours
    component: les1000jours
    owner: les1000jours
    team: les1000jours
  name: strapi-cache
  namespace: les1000jours
spec:
  rules:
    - host: backoffice-les1000jours.fabrique.social.gouv.fr
      http:
        paths:
          - backend:
              service:
                name: strapi-cache
                port:
                  name: http
            path: /
            pathType: Prefix
  tls:
    - hosts:
        - backoffice-les1000jours.fabrique.social.gouv.fr
      secretName: strapi-cache-crt
---
apiVersion: v1
kind: ConfigMap
metadata:
  name: strapi-cache
  labels:
    application: les1000jours
    component: les1000jours
    owner: les1000jours
    team: les1000jours
  annotations:
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/1000jours
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  namespace: les1000jours
data:
  nginx.conf: |-

    events {
      worker_connections 1024;
    }

    http {
      include mime.types;
      default_type application/octet-stream;
      sendfile on;
      keepalive_timeout 65;
      client_max_body_size 1024m;

      proxy_buffering on;
      proxy_cache_path /var/cache/nginx levels=1:2 keys_zone=small:128m inactive=1h max_size=5g use_temp_path=off;

      map $request_uri $nocache {
        \\"~*nocache\\" 1;
      }

      server {
        listen 80;
        server_name localhost;

        location /uploads {
          try_files $uri @api_backend;
        }

        location /graphql {
          try_files $uri @api_backend;
        }

        location / {
          proxy_pass http://strapi:80;
        }

        location @api_backend {
          error_page 418 = @no_cache;
          recursive_error_pages on;

          if ($nocache) {
            return 418;
          }

          proxy_ignore_headers Cache-Control;
          proxy_cache small;
          proxy_cache_methods GET POST;
          proxy_pass http://strapi:80;
          proxy_cache_key \\"$request_uri|$request_body\\";
          proxy_cache_valid any 1h;
          proxy_buffers 8 32k;
          proxy_buffer_size 64k;
          proxy_cache_use_stale updating;
          add_header X-Cached $upstream_cache_status;
        }

        location @no_cache {
          proxy_pass http://strapi:80;
        }
      }
    }
---
apiVersion: autoscaling/v2beta2
kind: HorizontalPodAutoscaler
metadata:
  name: strapi-cache
  namespace: les1000jours
spec:
  maxReplicas: 5
  metrics:
    - resource:
        name: cpu
        target:
          averageUtilization: 100
          type: Utilization
      type: Resource
    - resource:
        name: memory
        target:
          averageUtilization: 100
          type: Utilization
      type: Resource
  minReplicas: 2
  scaleTargetRef:
    apiVersion: apps/v1
    kind: Deployment
    name: strapi-cache
---
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: netpol-les1000jours
  namespace: les1000jours
spec:
  ingress:
    - from:
        - podSelector: {}
    - from:
        - namespaceSelector:
            matchLabels:
              network-policy/source: ingress-controller
    - from:
        - namespaceSelector:
            matchLabels:
              network-policy/source: monitoring
  podSelector: {}
  policyTypes:
    - Ingress
---
apiVersion: apps/v1
kind: Deployment
metadata:
  annotations:
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    kapp.k14s.io/delete-strategy: orphan
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/1000jours
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  labels:
    app: strapi
    application: les1000jours
    component: les1000jours
    owner: les1000jours
    team: les1000jours
  name: strapi
  namespace: les1000jours
spec:
  replicas: 1
  selector:
    matchLabels:
      app: strapi
  template:
    metadata:
      annotations:
        kapp.k14s.io/disable-default-ownership-label-rules: ''
        kapp.k14s.io/disable-default-label-scoping-rules: ''
        kapp.k14s.io/delete-strategy: orphan
        app.github.com/job: '5678'
        app.github.com/ref: refs/tags/v1.2.3
        app.github.com/repo: socialgouv/1000jours
        app.github.com/run: '1234'
        app.github.com/sha: '0123456'
      labels:
        app: strapi
        application: les1000jours
        component: les1000jours
        owner: les1000jours
        team: les1000jours
    spec:
      containers:
        - image: >-
            ghcr.io/socialgouv/nos1000jours/strapi:sha-0123456789abcdefghijklmnopqrstuvwxyz0123
          livenessProbe:
            httpGet:
              path: /_health
              port: http
            initialDelaySeconds: 30
          name: strapi
          ports:
            - containerPort: 1337
              name: http
          readinessProbe:
            httpGet:
              path: /_health
              port: http
            initialDelaySeconds: 30
          resources:
            requests:
              cpu: 5m
              memory: 256Mi
            limits:
              cpu: '1'
              memory: 1Gi
          startupProbe:
            httpGet:
              path: /_health
              port: http
            initialDelaySeconds: 30
          volumeMounts:
            - mountPath: /app/public/uploads
              name: uploads
          envFrom:
            - secretRef:
                name: azure-pg-user
            - secretRef:
                name: strapi
            - configMapRef:
                name: strapi-configmap
          env:
            - name: BACKOFFICE_URL
              value: https://backoffice-les1000jours.fabrique.social.gouv.fr
            - name: DATABASE_CLIENT
              value: postgres
            - name: DATABASE_NAME
              value: $(PGDATABASE)
            - name: DATABASE_HOST
              value: $(PGHOST)
            - name: DATABASE_PORT
              value: $(PGPORT)
            - name: DATABASE_USERNAME
              value: $(PGUSER)
            - name: DATABASE_PASSWORD
              value: $(PGPASSWORD)
            - name: DATABASE_SSL
              value: 'true'
      volumes:
        - persistentVolumeClaim:
            claimName: uploads
          name: uploads
      initContainers:
        - env:
            - name: WAIT_FOR_RETRIES
              value: '24'
          envFrom:
            - secretRef:
                name: azure-pg-user
          image: ghcr.io/socialgouv/docker/wait-for-postgres:6.56.1
          imagePullPolicy: Always
          name: wait-for-postgres
          resources:
            limits:
              cpu: 20m
              memory: 32Mi
            requests:
              cpu: 5m
              memory: 16Mi
---
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations:
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    kapp.k14s.io/delete-strategy: orphan
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/1000jours
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  name: strapi
  namespace: les1000jours
  labels:
    application: les1000jours
    component: les1000jours
    owner: les1000jours
    team: les1000jours
spec:
  encryptedData:
    MAIL_HOST: >-
      AgALfRwT4y4bsqEPKCbJY0WWxDDx4G4k7Pd2ixVC200likqiv3gwFVL3Qp1o1kxjLGL4rX2L5EA2J6/Vi/hAPGoGePSY4b+R1ckHcWGvNyYzJ0mKz0eaSkzVuRbhOfQ/10GeqCpPY7IkjcZFTb41RbL7apnYhOs25VJfgjRDGpqyf3A/ZpI8zLABM2aHJvqQw2p9HhJawOuzv4F5hC+E/UaGu3s2RwR/NYTYP06RW2DBxJqA5X+87uzK9QEamIJerkIWKxileNQUK+oxFlrcm4CIYMDKQ3x/mwnYskEeULBLXPkBkb4wYzXVcPjJ0xVgiSD9BaK6QoSSG/GELh1sQcA2bk/9IOBkIA1JGVoOkoVVZufMaVFFbrNW5nLlIzWAlzJqRuGt1Vbcw7YrTdYri12bBDn3jcFTB/aLUdT63MYBhxfI0394H46+vISNA5D+urGJGV0tecJbkJwOyaD8MMUww/qcO9pFsyjkPfQ/Fr3IWX9WsfzAdJQL9HhJBTrrjXeLzHjZEf8KGpU+Zfh84eV8zNjq5lzK6GyAKek9RySs/iVG47lruytxGirY4qIIp6Z5jZ9cM6sp2Dh0a+E+Evc76UFBQnon840O1rdI3PysivcPuRMKZfh0J9cQBcjuIVgXgt6ks90vD8R17cFapNnm2Nu6+DlQ25t08NkvJn+5iCEegVb5/eXm99MPDyzNMBJws7XPPdMcLMqSJMuttgmnbA==
    MAIL_PORT: >-
      AgBEpYbh/4DxVC55V4dT4px1BxTGnY+6cX7+T9fqfTTCXE4QzOOwugxVfVN4ecxQ1/W9uWtLin1WD8TsKeAd7lvf/EkHuWizIVP4mW0SQNUGQkN4lQ8sKOys0WlGQqvINs0OFGejAiqNX8fxMxVKCMYKsK/o3vTxbdscoaSAryhkDXyGK44RtP/lAhknpvU6zDrAleaBuFnvRWkz0mlOuUarQqRCtTJTWJ9zd6HmpvLmTfdvGWIrmE+2LVm8k4zhb3iZX+vXJJnCutiS4y/y2QetrCgJGQaXyUpnpJ5Xng+s+HAAkK2RCszp0CwDkMJjR9EIsmDGrpzN3AQb4DE/AVWWoo3naS53YuFhU2wGFwtzh6s04jn4/1hJBNi726bVDWzF9mvstf/7dtjHvrd1Eg2wZTKLxHFZ6fF9BQM5Ql7uHoEVE/N0RM5pKY9aPF2CRgQiPT29STAHqRjgIbrv4K/a++9AoTAavFp8JoSyAf8h4UCS3MCPonv4N9THy002eE0tvw7ysu19hoOsFLoQfcj2dbq1ssXPU9oHps/RP0aFQ8kn9bY0NbxXR4mIQlbkjExhPtUkog6eC7lGEqbsUsrc9GSDf21SUMkwHG599IUTQC3JLIwJvgt5Oiu3l1NaRHblnzICsdCEBzEwBDXVNi42cmeDINlmTQN9Kg2AbOm8YaW4WACBGuR9U40+ac2KqXd3s9k=
    MAIL_USER: >-
      AgBhhLXEYmJt04ZxP7NK1GQvo/LdIQKvhZoFBwwwj5MGUBaqnKLxmA09wZyBmSTerISlT1sFE+fjlMlb6kVBVaC2xRS0Qan9dfNDo8/k6kz1+M5O8gxvfKpSwCysdi9q0LlY+uoBrsZ124ubHbHr2UG5By1XfbIfhe9AwvsnN6JP+1IIcftN56UI2KM00Lob2/d4TsmoM6u4pvjq0cP3r5v89jPQ3lNKq7V2IeCS9OJbyFpzOBF+HTCjAukIsEMVg/vrM4Dj4YyRMwwOdWyANHDpe+pFtNfL1LD8JOVI4anTuVVLx2u9d2VWzK+J2I/u7QviwVrKfgumkZtkWYasrMsnMH5WNp2RyhqhiotApLmfLSoHzVOOuqd6dOMGZJ+14fyI8HMpvIlVnaKizGPe8PnVPiDwx0Umn0A+FkW1ssBlKdqAhu3CZaTv8JBn5gBFlOCxiyARt934bPlW4I7xUWxqiM1XTCHc7rDPGGaldtpcjYCe+YzLQ6GHul+UT8BLTbaRWcV72l4ZU70e3Ar3hwXdRdHsb0KLrWHj4/R7/9lmaXhQpp82/Wgvyvwk44pqEv/6aZFDLcTjcEYtET7zK88/8WDrHWBCVUG6HFx5/WNybZeLoWA7CA0GY6G1YKFU1/v7CLGlq0eQnuNjYNsSVT3UtVWefDSWuxqHXVsK+oBeqzEyUIhaJWkltvgsaicWZUZEI88C8vd1t8Yskj/t0q8WpCDyCXu6zweQZPnp26kOnw==
    MAIL_PASSWORD: >-
      AgA2ysSWEdrwvipdmBH5nKoZ/JXKaBi5hPtdr9fV3btJLom5nwSyDavb+yshBCsqkeUGm56AR51MpB0Z0BZ1pjgNKrSFMLtW+yqR75qrjSjsLXmrI4HktEreBrYn/mL3GMV/eAeyzi5dteIxxYfku0KK4+UIcpKsOVHztx+J2jm0RMHFvHIwXgv8XXObKmXhRimBHqGgiqWM1kkyvZqFrqVqxPeIQSg1wips4a0nsfzzqABR7sAm/d5jnsX1ln2dRqy5OojSTUPseRS+xlZWfHMz+qW6FHCRHlcVMsTK0TatNBZwlHtjoIact7G11OjyCA5aIRAQDxCefDEiJHgj0pxFQio/rt7R3boCLK5eLEcpkQ9AQVVjdLZ02fHT/6v+PAbxkrESUmaCuBBTwku+2cGx6aCFnqiO5O3yd6AADvcIstGKuujvGqvB+xSiR2Kckw2oaYZHMvIDx/C5yADSUqA8ffY7cGJj0i5CLC928+0/C8Fud0GGc1oGz3y0vpzY93xjNY90qRRAoSyjJiU7IokYB15+gAjaU1RYgmTQWQV0OfQDrIV5FLu3PJ3s7TvcHrWYD6myi8rI4llhYaPgIrC2GArCj1A2CZ2gYlr1zEAVVu9/X3N/NZQCbmOCz3poI/+KaLO7yndM8Rsbi2gnyWAknkK/NX0j8vCB/fJP4Y8uWB7GWrAWpSk6EgfmGjMHdhU0sk+rkiCLtgDR0dSUYwiCXonD74XuYZQ3nWbGa19h4w==
    MAIL_SEND_FROM: >-
      AgAmOrfmRsQus9ac+6loOg2/EJcPIJqi0XcMgyQqwf8GAXR3sd957rLNHwYiButsLHhHql1L4jLScjxzJHc8+mVOC4NSBRpWx1xwRzMqZxVAEcPD0dH2TM5dfxWecyR2cG/gvArM/QnEj/glt6QiUMkl9ydF3xfgkAMNNjYj+vlDvwcOVsiS17eIbsEe37xhdZ5wMkzSCn+jrV/pamiCMNRpTX6enarM7cSkN23Ix3KdvyzYTN8uC83SvJY1UFT7zQOL49u3+6IazQZtyd/9sePlrLMosl13c9spWhvZoDG1QlrhqJGBb1NDtaG5HGcb7UJbDBLnlt7MEYubaTmzVhQX8Ckpb9dBlUt69M5VwVq3d3CWAIT/9SR+s/IQt52HbCOq95iXu5mon4qR9cpIDkaNFERSRpQ9kuoJ0mMp3ZqkErlg05cPuQgpLAxpoTqPOz/bRhD8EGWWha957mcMusFAOVxH16UutqIV+dRPQaiPmvnq9/nk17+xGy4vnrrr55V/h2oy9tl6ivRa03dh97L4upn/0n/1yekMME6pi5anFh875KvZO7n/tUI1HY+9tMhQ5yHg6MaHyCcAGqn24otNLkuNpefSt1CA9whdDvqHj/PX2MeDdkvTB3vCp6ljxj/kcb+D4fJCeyr9EcjCbCPGLpxDVgu8PYX1+QbnwrFRKTHhIc7Q35qZARai/hwJCIoML1j4YbTxzr5NDfsox3mS/4kV+r9Rw896a5V12aqzuT2b0f9praYWRW1JEw==
    MAIL_SEND_TO: >-
      AgB/oa7mVWCebUR9mSYMA7YaQANq/8St8x5IpOcE5gYjdKO+pVyf65Tp49h3oG2r6xPuXN3rUBDLUUqen5vZGwzARSSxS9YwdM7eJq3vXMsrVj0otqSfOoi+L9fc+9C99WOMkSG8bSmFSGtmzdmbzEGkwXKs+KyYRXTjV4J1r3nCZIJi06ifMKj2RTarnHlcTTCLqdltnRaT3B8Smo/0UaIu6GVLKMA2Ag1QDiHG3jLVIJJi5dukNOoM5tB3Qdo8UgGSWTOQ1UlRl/X5P6gVV8jN/cgRJ3D3IYxi8cHwvC5gnnRsg8zWNSdqWCsNbxb34ie3bvssG18JANPu5KCjwVhmhL5BZmb+UOTMBY4klPR5yrpu6zfeLyMr6FHsgOlFem7OdGEVbFgA0pWpJpaih0xwlWxYojadHXZRyiRASJaGlxUI0Pdw77WXFDF0f9vRWaN4UQMx3D4CGwuFNEDIqJztXXCy1ZeqlMky4HRCC9ecT8/KwcvGmYd7exUtd5U4XHk+0j9qcG4cF7l/I1nw4NuTdfLdo89wkghxSWpkyYCp2N/b0Ya+yOXQFmty+UQLY2eLDJUwjRTcEFLGL/fVcuRwrB9b1Wpv7FVQFEVsiTLKD6vS6MLksucI1kPgUmiYeaRQ0u6N98vuokWm5W6U+K1ekPmjxPwf3SdvQZAaEACzch1SVBR1iQpPSssB3/1NUPrbPf9zU8Ii65c+N63iR1Kl06ZF+H1Jek+S4YlZClnFYN/B1n82eA==
  template:
    metadata:
      annotations:
        kapp.k14s.io/disable-default-ownership-label-rules: ''
        kapp.k14s.io/disable-default-label-scoping-rules: ''
        kapp.k14s.io/delete-strategy: orphan
        app.github.com/job: '5678'
        app.github.com/ref: refs/tags/v1.2.3
        app.github.com/repo: socialgouv/1000jours
        app.github.com/run: '1234'
        app.github.com/sha: '0123456'
      name: strapi
      namespace: les1000jours
      labels:
        application: les1000jours
        component: les1000jours
        owner: les1000jours
        team: les1000jours
    type: Opaque
---
apiVersion: v1
kind: ConfigMap
metadata:
  name: strapi-configmap
  namespace: les1000jours
  annotations:
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    kapp.k14s.io/delete-strategy: orphan
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/1000jours
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  labels:
    application: les1000jours
    component: les1000jours
    owner: les1000jours
    team: les1000jours
data:
  STRAPI_LOG_LEVEL: info
  NODE_ENV: production
  SENTRY_DSN: https://5dbac413000f4a43a073841acebc82f5@sentry.fabrique.social.gouv.fr/55
  RESOURCES_URL: >-
    https://nos1000jours-web-pro.fabrique.social.gouv.fr/annexes/ressources-parents
---
apiVersion: v1
kind: Service
metadata:
  labels:
    app: strapi
    application: les1000jours
    component: les1000jours
    owner: les1000jours
    team: les1000jours
  name: strapi
  annotations:
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    kapp.k14s.io/delete-strategy: orphan
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/1000jours
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  namespace: les1000jours
spec:
  ports:
    - name: http
      port: 80
      targetPort: 1337
  selector:
    app: strapi
  type: ClusterIP
---
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations: {}
  name: azure-les1000jours-volume
  namespace: les1000jours
spec:
  encryptedData:
    azurestorageaccountkey: >-
      AgB/Hb0mdnKqiNIho6AaV4mbvdurMoyHgTbSw8BNrWrdz5GEjHWRNU09O/8NVMZ4GJNzC/ZLyOQLMlaA1YEfYYxkWsq6loe+Kd4AzSDGG1eKNvvOdlt0G1Me8Ou0jEe9SM0lkcmBFA8O9BXuJVoHsS8LI8f+gMGlU1qAOnvo67TEqXTktp2u9Afej6Y280OFBUVOHBg1SbEib0Jxs0nvxfEdxpJK66VSP9wrPWm20rhT0VeVwVLqcjUSeMv3l/755IagNFq9M1I1YJ9QyT5gbGe4zZKCVQGnfRAu+pUJnIQQI6hOxjzEqbMDgGUoqg9ivsrqu4fsffG6vfJk6SJ/QK3yWvQfWhuj/rTQmRIcdBaSguyVCCrrcwzPs8Ahu06Nof3GCy/eOgPibzFgNKg74s3wrmoNLynjcJmR//HTHOtv8tORKyxAelHO0PauZN0meqjtLqLLrZbv8pmfBbo5hzXsS7D/+C+1rE/5Nqmln6sLki2eUrs1pSDAoibwv9Ho+RqGkz/TCsT5ZmpD41C0bJQQtsdwB5sym4XwI3w9RxNnKJ2shSqDGH5NIA1GCiH1R2ek+zsHgAe7RdRWgW7e7nYoq0jzN/WM1LPXAaqgFoYXw71zlUHRMxrXaiN2S+/1WC3pMpwbOL8dIIbaPG0C0bnK0zWe8RIIJwqCo/qyrX6RflQ2q2j6zYnEeQ5SpyNPJXgtd4iKQzr+n3TO8XG945MkyxLDtJQfPSuVoc7iXMVndbeZIrBeSNV+cAd7fmtSIM3ENnXGDZJyKp19tcn1coud4Pf2E6BTbuEAqgJiYvjHVj4nGGNOqf7l
    azurestorageaccountname: >-
      AgB0GacvV3c3G5PL92GsgzVLjS5MrU5zHM2P6SjtMq1ax8tjAKM8WOhh951J+xbLlr1DHLv2Fp+ExYuoWD9KjaA4Irq/b+n8uluVML7iUYWyfcZ4D+Q7NV9YMh+Cw2zBEY/+wtPBv3xIJl77ckq27DaAnUoBDQslajGJX6+jpN5MhVR61fxA74+WZFKzHZ+oM0yW7AuFMpgqG5vI0ggiJOqoMDWCmNdsd5GxTJARK8x4YlhgqsduUKKzWK9VkPFclUX6W8YyY2ndjfod6TE3REOhSlCO6kBPDx5wLaF+l0//C0fGnf6euKU4jAza9gtglHx70Fhui3+oQqCco9jYv1AEo90qvRbLKcNSfGhEEs2E7PSqNWj2g4jt/VsYXFC/NtELstwqjqSHCTl0BLBg8rlNQg3QX96aDvMWmGx02aR11JWxlln6uBbLNm66t/XL7KPWEtWZgAARq8sRqG19G8fK3OFqFahZA2LKAr2DEpu9L68a6OQM6cFL+9d0TTo+6PHJD2jCh+0bWvKihp7DzWT5FggHWzei5eRQjpbUI0rQVAsH2BxPbV6T51IBKTPpaJM6/dTMsCu4kFXcRxdxvzovJWPS7Yxe+1lHKfDFzJX+2oExHCDxjSPbj1n+VOsaoATJNTeluRFKLeNNTYiNjwxBj1oyPqybnbV6X5MdMxE6Zc12w9oLxhXM1GsKqzZI3rZmBUW8DW94jSj0R/tuLOBo
  template:
    metadata:
      annotations: {}
      name: azure-les1000jours-volume
      namespace: les1000jours
    type: Opaque
---
apiVersion: autoscaling/v2beta2
kind: HorizontalPodAutoscaler
metadata:
  name: strapi
  namespace: les1000jours
spec:
  maxReplicas: 5
  metrics:
    - resource:
        name: cpu
        target:
          averageUtilization: 100
          type: Utilization
      type: Resource
    - resource:
        name: memory
        target:
          averageUtilization: 100
          type: Utilization
      type: Resource
  minReplicas: 2
  scaleTargetRef:
    apiVersion: apps/v1
    kind: Deployment
    name: strapi
---
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: uploads
  annotations:
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/1000jours
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  labels:
    application: les1000jours
    component: les1000jours
    owner: les1000jours
    team: les1000jours
  namespace: les1000jours
spec:
  accessModes:
    - ReadWriteMany
  resources:
    requests:
      storage: 5Gi
  selector:
    matchLabels:
      usage: les1000jours-uploads
  storageClassName: ''
---
apiVersion: v1
kind: PersistentVolume
metadata:
  labels:
    usage: les1000jours-uploads
    application: les1000jours
    component: les1000jours
    owner: les1000jours
    team: les1000jours
  name: les1000jours-uploads
  annotations:
    app.github.com/job: '5678'
    app.github.com/ref: refs/tags/v1.2.3
    app.github.com/repo: socialgouv/1000jours
    app.github.com/run: '1234'
    app.github.com/sha: '0123456'
  namespace: les1000jours
spec:
  accessModes:
    - ReadWriteMany
  azureFile:
    secretName: azure-les1000jours-volume
    secretNamespace: les1000jours
    shareName: uploads
  capacity:
    storage: 5Gi
  persistentVolumeReclaimPolicy: Delete
  storageClassName: ''
"
`;
